### PR TITLE
[Cult Balance - March 2019] Tattoo Tweaks, Arcane Knowledge replaced with Arcane Dimension

### PR DESCRIPTION
**Pull request by DeityLink**
tattootweaks

:cl:
* tweak: Rapid Tracing is now a tier 3 tattoo (thus available at Act 3 once more). Speed bonus upped from 60% to 66% (1 less tick per word basically)
* rscadd: Arcane Knowledge rune is now Arcane Dimension, and tier 2. Lets you store an Arcane Tome into an invisible dimensional pocket and pick it up whenever.
* tweak: "Editors note - These cult updates won't be applied until all are ready."

### CODE REVIEW

**Comment by MadmanMartian** [comment]
Why's this one aimed at bleeding-edge?

File: code/datums/gamemode/factions/bloodcult/bloodcult_spells.dm [hunk 1]
```diff
@@ -61,16 +61,12 @@
 	var/obj/item/weapon/tome/A = null
 	A = user.get_active_hand()
```
Comment (MadmanMartian) [inline code/datums/gamemode/factions/bloodcult/bloodcult_spells.dm]: Remove this, the tome = istype(A), the if(!tome), everything held within that if(!tome), and replace it with var/obj/item/weapon/tome/A = locate() in user.held_items

File: code/datums/gamemode/factions/bloodcult/bloodcult_spells.dm [hunk 2]
```diff
@@ -239,3 +235,62 @@
...
+		stored_tome = null
+		name = "Arcane Dimension (empty)"
+		connected_button.name = name
+		desc = "Cast while holding an Arcane Tome to discretly store it through the veil."
+		hud_state = "cult_pocket_empty"
+		connected_button.overlays.len = 0
+		connected_button.MouseExited()
+		return
+
+	var/obj/item/weapon/tome/held_tome = user.get_active_hand()
```
Comment (MadmanMartian) [inline code/datums/gamemode/factions/bloodcult/bloodcult_spells.dm]: Again, var/obj/item/weapon/tome/held_tome = locate() in user.held_items, then remove the if(!held_tome)

**Review by MadmanMartian — CHANGES_REQUESTED** [review]

**Comment by DeityLink** [comment]
@MadmanMartian done

**Pull request merged**